### DISCUSSION TITLE
posts/linux-kernel-pwn-06/

### DISCUSSION THREAD

**Discussion by giscus** [brant-ruan/fernweh-comments#7] [Announcements]
# posts/linux-kernel-pwn-06/

Linux Kernel PWN | 06 DirtyCred

https://blog.wohin.me/posts/linux-kernel-pwn-06/

**Comment by brant-ruan** [comment]
Question 1: What is the range of kernel versions in which DirtyCred can be used to exploit vulnerabilities?
